app/Http/Controllers/SensorController.php

```diff
@@ -70,10 +71,15 @@ public function store(Request $request)
     public function show(Request $request, $id)
     {
         $sensor = Sensor::findOrFail($id);
-        
-        $sensordata = $sensor->data()->paginate(15);
+        $sensordata = $sensor->data()->paginate(25);
+        $chart = Charts::create('line', 'highcharts')
+            ->title($sensor->name)
+            ->elementLabel($sensor->type)
+            ->labels($sensordata->pluck('created_at'))
+            ->values($sensordata->pluck('value'))
+            ->responsive(true);
 
```

Comment:
Can you keep this order for the graph but reverse the order for the sensor history panel. That way users can see the most recent sensor data in the history panel first.

Reply:
Fixed in 2c99763